packages/component/src/Activity/Timestamp.js

```diff
@@ -20,13 +22,15 @@ const Timestamp = ({
   </span>;
 
 Timestamp.defaultProps = {
+  'aria-hidden': true,
```

Comment:
Also, please default this to `false`.

```suggestion
  'aria-hidden': false,
```